contracts/truefi2/TrueFiCreditOracle.sol

```diff
@@ -20,6 +20,9 @@ contract TrueFiCreditOracle is ITrueFiCreditOracle, UpgradeableClaimable {
     // @dev Track credit scores for an account
     mapping(address => uint8) score;
 
+    // @dev Track credit scores for an account
+    mapping(address => uint256) borrowLimit;
```

Comment:
Could you change this to `maxBorrowerLimit` to match the spec? `borrowLimit` by itself could confuse users and give them the impression they could borrow up to that amount.

Also don't forget to update the comment!